fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1420,784 @@ def draw_act(self, obj, layout, value):
         layout.prop(obj, "sm64_obj_use_act" + str(value), text="")
 
 
+class AddBehavior(bpy.types.Operator):
+    bl_idname = "bone.add_behavior_script"
+    bl_label = "Add Behavior Script"
+    option: bpy.props.IntProperty()
+
+    def execute(self, context):
+        prop = context.scene.fast64.sm64.combined_export
+        prop.behavior_script.add()
+        prop.behavior_script.move(len(prop.behavior_script) - 1, self.option)
+        self.report({"INFO"}, "Success!")
+        return {"FINISHED"}
+
+
+class RemoveBehavior(bpy.types.Operator):
+    bl_idname = "bone.remove_behavior_script"
+    bl_label = "Remove Behavior Script"
+    option: bpy.props.IntProperty()
+
+    def execute(self, context):
+        prop = context.scene.fast64.sm64.combined_export
+        prop.behavior_script.remove(self.option)
+        self.report({"INFO"}, "Success!")
+        return {"FINISHED"}
+
+
+class BehaviorScriptProperty(bpy.types.PropertyGroup):
+    expand: bpy.props.BoolProperty(name="Expand")
```

Comment:
Default to true would be better in my opinion

Reply:
When adding in a preset behavior script, which I think will be the normal usage, it will add in like 8 or so cmds at once all open. So having it default closed will be better to avoid bloat. If you use the operator to add in a new cmd I will instead make it so that single new one is open.

Reply:
That sounds good to me